pkg/common/telemetry/names.go

```diff
@@ -175,9 +175,15 @@ const (
 	// BySelectors tags selectors used when filtering
 	BySelectors = "by_selectors"
 
-	// CallerAddr labels an API caller address
+	// CallerAddr labels an API caller address in host:port format
 	CallerAddr = "caller_addr"
 
+	// CallerIP labels an API caller IP address
+	CallerIP = "caller_ip"
```

Comment:
We could repurpose `CallerAddr` to mean IP and introduce `CallerAddrPort` to replace the current `CallerAddr`, but that would introduce backward incompatibility. I'm not sure `CallerIP` is the best naming either.